src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -4,9 +4,11 @@
   "Actions": "Actions",
   "Activity": "Activity",
   "Add": "Add",
+  "AddAutoTag": "Add Auto Tag",
+  "AddCondition": "Add Condition",
+  "AddNew": "Add New",
```

Comment:
This same change is made in a different PR already... https://github.com/Sonarr/Sonarr/pull/5837/files#diff-747fe3f19f5191583bc9eb40ab108ebb89e05e498c98398aba00ef9e5295f120R7